Detectors/TPC/base/include/TPCBase/PadSecPos.h

```diff
@@ -6,20 +20,25 @@
 
 namespace AliceO2 {
 namespace TPC {
-  class PadSecPos {
-    public:
-      PadSecPos() {};
-      PadSecPos(const Sector& sec, const PadPos& padPosition) : mSector(sec), mPadPos(padPosition) {}
-
-      Sector  getSector() const { return mSector; }
-      Sector& sector()          { return mSector; }
-
-      PadPos  getPadPos() const { return mPadPos; }
-      PadPos& padPos()          { return mPadPos; }
-    private:
-      Sector mSector{};
-      PadPos mPadPos{};
-  };
+class PadSecPos
+{
+  public:
+    PadSecPos() {};
+
+    PadSecPos(const Sector &sec, const PadPos &padPosition) : mSector(sec), mPadPos(padPosition) {}
+
+    Sector getSector() const { return mSector; }
+
+    Sector &sector() { return mSector; }
```

Comment:
I believe even for the non-const access, we should call the function "getSector()"